plugins/woocommerce/tests/e2e-pw/tests/admin-analytics/analytics-data.spec.js

```diff
@@ -25,7 +25,7 @@ const test = baseTest.extend( {
 
 test.describe(
 	'Analytics-related tests',
-	{ tag: [ '@payments', '@services' ] },
+	{ tag: [ '@payments', '@services', '@skip-on-default-pressable' ] },
```

Comment:
Skip Analytics-related tests since there are already some numbers on Pressable and the sum at the end is not the same.